mesop/web/src/component_renderer/component_renderer.ts

```diff
@@ -210,7 +210,7 @@ export class ComponentRenderer {
         // We should have checked this in Python, but just in case
         // we will check the attribute name right before using it.
         checkAttributeNameIsSafe(key);
```

Comment:
Do we need to check attribute name still if we're using properties?

Reply:
Also above there is boolean check. Does that apply to the properties change as well?

Reply:
good questions - I'm not 100% sure if the attribute name check is still needed. When I experimented with it, things like `onclick` which are unsafe as an attribute (e.g. evals a string as a JS function) don't seem to have the same issue as a property, but I'm not sure if this is true across all browsers. Also the src* checks still seem valid for property. Since it's always easier to remove a check and I haven't heard anyone complain yet about these checks, I'll leave in the name check.

The boolean check, though, I think we can remove - I tested it out and I don't see an issue with always setting the boolean property.